### DISCUSSION TITLE
求助!

### DISCUSSION THREAD

**Discussion by LuAn-afl** [share121/inter-knot#3239] [其他]
绝区零建筑物人物模型等穿模怎么办本人手机是IQOO7刚开服的第三天就出现这种情况了，反馈也没用，后面一直用另一个手机玩的，今天这个手机下回来发现还是穿模

https://github.com/user-attachments/assets/aedf124d-a128-4ab7-b4b8-7e9f239e0816

**Comment by LT-UT** [comment]
没看到啊，右键显示播放音频
就只有声音
可能还没调试完，将就玩吧<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3d389fd2885b008e46d843c552c02bf3_5073438697399152551.png" alt="开拓者 吃瓜脸" width="75" >

**Reply by LuAn-afl** [reply]
真的将就不了啊全穿模了打怪怪也穿模模型消失
